script/Ciel/Day 12/12_01_CIEL12_2B.txt

```diff
@@ -397,27 +421,34 @@ Tohno-kun, is that really how you saw me?!"
 [sha:5686a1204a3071a65046da59fe37c00602a4b8d0]{
 -- Page 57, Offset 31010.
 -- ……しまった。微塵も思っていなかった、と断言できないのが我ながら薄情だ。
-...I messed up. The fact that I can't completely deny it is heartless, even for me.
+I messed up...
+ The fact that I can't bring myself to pretend I didn't mean it is a little heartless, even for me.
```

Comment:
"I messed up..." could use a replacement. It feels too descriptive, and not evocative of his feelings.

>Shouldn't have said that.

>Damnit...